wurst/_wurst/Orders.wurst

```diff
@@ -0,0 +1,371 @@
+package Orders
+// Credits Shadow Daemon for cj_order.j from cJass
+import NoWurst
+
+/** Class that contains every known order in game that has no string counterpart */
+public class SpecialOrders
```

Comment:
Is there anything else special about them? 
Maybe you could make another class containing UI order (move, attack, smart)

Reply:
There's nothing more than it really. It was the first name that came to my mind and I don't really care about it since it is included into Orders class, so feel free to suggest something else.
Well, it intends strict hierarchy if I want Orders class to extend every other class with orders, so I thought: if it is not usual orders, then it should be some *special* orders.